README.md

```diff
@@ -9,15 +9,23 @@
 
 ---
 
-Our templates offer a rich boilerplate to jump start React-based application development for [Create React App](https://github.com/facebook/create-react-app).
+<p align="center">
+  <a href="https://www.npmjs.com/package/cra-template-nimble"><img src="https://badgen.net/npm/v/cra-template-nimble" /></a>
+  <a href="https://www.npmjs.com/package/cra-template-nimble"><img src="https://badgen.net/npm/dy/cra-template-nimble" /></a>
+</p>
+
+Our template offers a rich boilerplate to jump-start React-based application development with [Create React App](https://github.com/facebook/create-react-app).
+
+## Getting Started
+
+### Prerequisites
 
-## Get Started
+[![node-version-image][node-v-src]](https://nodejs.org/download/release/v14.18.1/)
 
-### Use the template
+### Usage
 
-To use this template, add `--template nimble` when creating a new app.
+To use this template, add `--template nimble` when creating a new app from the `create-react-app` package.
```

Comment:
not sure if it should be called package 🤔 I feel like a 'command'
as a whole when we add the template params

```suggestion
To use this template, add `--template nimble` when creating a new app from the `create-react-app` command.
```

Reply:
True, the package name is `react-scripts` (which contains the config associated with the cra command).
And in the context of "when creating a new app", it definitely refers to the command ;-)

Reply:
✅ 9687b2c